docs/source/tutorial/python/featurizers/molecule_featurizers.ipynb

```diff
@@ -118,13 +124,160 @@
+       "       5.99833786e-02, 9.05498589e-08, 4.60978367e-10, 1.57072376e-01,\n",
+       "       1.66847964e-01, 2.37240682e-02, 8.07601514e-02, 2.75008841e-02,\n",
+       "       4.92845505e-03, 1.24459630e-01, 7.31816496e-02, 1.67096874e-01,\n",
+       "       7.55810089e-02, 8.78622233e-24, 1.33643046e-01, 3.04494668e-02,\n",
+       "       2.58369311e-02, 5.30138094e-05, 1.42657565e-16, 3.73160396e-02,\n",
+       "       6.95272017e-13, 0.00000000e+00, 9.79690873e-13, 2.64281353e-04,\n",
+       "       1.20493060e-11, 2.86305006e-09, 1.04578852e-01, 3.09944928e-02,\n",
+       "       2.99487758e-06, 2.77639012e-01, 5.30138094e-05, 6.17138309e-03,\n",
+       "       5.30138094e-05, 5.00000000e-01, 3.84710451e-01, 5.30138094e-05,\n",
+       "       5.30138094e-05, 1.64664515e-01, 5.30138094e-05, 9.98653446e-01,\n",
+       "       3.99820633e-01, 2.02868342e-02, 5.70867846e-19, 3.32362804e-10,\n",
+       "       9.64197643e-10, 7.10542736e-15, 5.83707586e-13, 1.19880642e-20,\n",
+       "       1.65079548e-01, 1.67040631e-01, 1.66498334e-01, 1.66486816e-01,\n",
+       "       2.02864661e-01, 6.93658809e-02, 7.10542736e-15, 1.68346480e-01,\n",
+       "       1.67982932e-01, 6.87189958e-10, 1.18157291e-03, 1.64332634e-01,\n",
+       "       8.37776917e-04, 1.66325734e-01, 1.63034142e-01, 1.65079548e-01,\n",
+       "       9.56970492e-08, 3.49708922e-08, 1.68206175e-01, 1.65806858e-01,\n",
+       "       1.67346595e-01, 7.13964619e-07, 2.64115098e-12, 9.99127911e-02,\n",
+       "       2.86809243e-10, 3.77737848e-01, 4.50616778e-03, 1.33250251e-01,\n",
+       "       3.47299284e-02, 1.61482916e-09, 1.87517315e-18, 2.09410539e-07,\n",
+       "       7.10542736e-15, 4.99264281e-01, 1.64929402e-01, 1.31744508e-17,\n",
+       "       2.11164355e-16, 1.16815875e-09, 3.25923600e-22, 6.24601420e-10,\n",
+       "       1.68149182e-01, 1.65450729e-01, 1.17110262e-13, 0.00000000e+00,\n",
+       "       1.64668868e-01, 1.66924728e-01, 0.00000000e+00, 5.10071327e-08,\n",
+       "       7.10542736e-15, 1.54654108e-01, 2.79420938e-22, 0.00000000e+00,\n",
+       "       1.67639733e-01, 6.31499266e-25, 1.68186130e-01, 9.08850267e-03,\n",
+       "       1.68363202e-01, 8.26542313e-11, 1.56346354e-01, 0.00000000e+00,\n",
+       "       0.00000000e+00, 2.11354236e-02, 2.11354236e-02, 2.38815575e-20,\n",
+       "       0.00000000e+00, 8.33672450e-25, 5.30138094e-05, 1.56951066e-01,\n",
+       "       4.03434503e-08, 1.55259196e-23, 1.59306117e-17, 5.76610077e-14,\n",
+       "       2.95798941e-11, 1.68378369e-01, 1.67380186e-01, 1.48151465e-18,\n",
+       "       2.32414994e-16, 4.70359809e-08, 1.66633397e-01, 1.87492844e-01])"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "molecule_featurizer = V1RDKit2DFeaturizer()\n",
+    "molecule_featurizer = V1RDKit2DNormalizedFeaturizer()\n",
```

Comment:
Is the first line intended to demonstrate the availability of this featurizer? Adding another cell to show the difference might be overkill. I'm fine with keeping it as it is now.

Reply:
Yes it is intended to demonstrate the availability